dpnp/tests/test_mathematical.py

```diff
@@ -1350,7 +1350,11 @@ def test_1d(self, dt):
 
         result = dpnp.i0(ia)
         expected = numpy.i0(a)
-        assert_dtype_allclose(result, expected)
+        # NumPy promotes result of integer inputs to float64, but dpnp
+        # follows Type Promotion Rules
+        skip_dtype = [numpy.int8, numpy.int16, numpy.uint8, numpy.uint16]
+        flag = True if dt in skip_dtype else False
```

Comment:
we can make it shorter:
```suggestion
        flag = dt in [numpy.int8, numpy.int16, numpy.uint8, numpy.uint16]
```